app/Http/Controllers/PostController.php

```diff
@@ -26,16 +26,12 @@ public function store(Request $request): RedirectResponse
             'title' => ['required', 'min:1', 'max:30'],
             'body' => ['required', 'max:250'],
         ]);
-
         $post = new Post;
         $post->title = $request->input('title');
         $post->body = $request->input('body');
         $post->user_id = auth()->user()->id;
-
         $post->save();
-
         session()->flash('status', 'Post Created!!');
-
```

Comment:
Cheo estas líneas sí deben dejarse. Revisa este enlace que tiene una explicación de los motivos
https://spatie.be/guidelines/laravel-php#content-whitespace

También por el estándar PSR-12, en él definen que las líneas antes y al final de bloques mayores de código: definición de clases o métodos, finalización de clase, entre otras, son los lugares en donde no deberían dejarse líneas vacías.

> 2.3 Lines
There MUST NOT be a hard limit on line length.

> The soft limit on line length MUST be 120 characters.

> Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

> There MUST NOT be trailing whitespace at the end of lines.

> Blank lines MAY be added to improve readability and to indicate related blocks of code except where explicitly forbidden.

> There MUST NOT be more than one statement per line.

https://www.php-fig.org/psr/psr-12/